tests/components/blebox/test_climate.py

```diff
@@ -254,3 +279,53 @@ async def test_update_failure(saunabox, hass, config, caplog):
     await async_setup_entity(hass, config, entity_id)
 
     assert f"Updating '{feature_mock.full_name}' failed: " in caplog.text
+
+
+async def test_reding_hvac_actions(saunabox, hass, config, caplog):
+    """Test hvac action for given device(mock) state."""
+
+    caplog.set_level(logging.ERROR)
+
+    feature_mock, entity_id = saunabox
+    await async_setup_entity(hass, config, entity_id)
```

Comment:
Setting up the integration like this seems not needed:
https://github.com/home-assistant/core/blob/aa4a0df194d7d728b171a2c0bf31bedc4e4fa2df/tests/components/blebox/conftest.py#L84

Nothing will be done with `config`. This integration uses config entries.

Set up the MockConfigEntry instead with `hass.config_entries.async_setup`.